test/errors/incompatibleTypeAndInit-detailed.good

```diff
@@ -0,0 +1,9 @@
+─── error in incompatibleTypeAndInit.chpl:1 [IncompatibleTypeAndInit] ───
+  Type mismatch in declared type and initialization expression.
+  In the following declaration:
+      |
+    1 | var x: int = "hello";
+      |        ⎺⎺⎺    ⎺⎺⎺⎺⎺⎺  
+      |
+  the type expression has type 'int(64)', while the initial value has type 'string'.
```

Comment:
Maybe "type expression" -> "type specifier/specification"?  Or "type constraint"?  (since it may be a partial constraint in some cases like `var x: integral = "hello";`?)